pages/linux/sslstrip.md

```diff
@@ -0,0 +1,24 @@
+# sslstrip
+
+> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks. Perform an ARP spoofing before using this tool.
```

Comment:
```suggestion
> Hijack HTTP and HTTPS traffic on a network
```

You will have to correct me if this is wrong, I am unfamiliar with it.

However this description fits the style of a description

Also with the ARP spoofing suggestion, it may be better suited on its own line.

such as 

```
> Run a ARP spoofing attack in conjunction.
```

Reply:
Thanks for the suggestion.
sslstrip does not hijack http/https traffic,  it basically decipher https traffic by downgrading the encryption protocol to the insecure one, so I will stick to the original description.

For the second suggestion, I would say we need to run ARP spoofing first to hijack traffic first. "In conjunction" may be a bit misleading.

Reply:
IMO, I prefer current description than the suggested one: "Hijack HTTP and HTTPS traffic". 

If the users want to learn more about this command, current description provides the key word to find related resources.